### ISSUE TITLE
Pre-trained model files

### ISSUE THREAD

**Issue by amanshenoy** [donydchen/FMPN-FER#9]
Hey, I was wondering if the pre-trained models are available and if so where I could find them?

**Comment by Cip0** [comment]
Did you find them? I am looking for them as well.

**Comment by mercy-thuyle** [comment]
Did you mean the output of pre-train in strep "_**Pretrain the Facial-Motion Mask Generator (FMG)**_" ?
